roles/firewall/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: Install 'ufw'
```

Comment:
Keep the style, lowercase and don't quote `ufw`.

Reply:
done